Trio/Sources/APS/APSManager.swift

```diff
@@ -668,6 +669,20 @@ final class BaseAPSManager: APSManager, Injectable {
             return // return without throwing an error
         }
 
+        // Directly check if the pod is faulted
```

Comment:
Let's add a `FIXME` comment to this change please. It looks okay, it works (tested in-vitro), but this goes so deep into a specific pump manager's notification behavior, this seriously should be handled in a different way. We will have to revisit pump manager notifications and interfacing on a larger scale for that though.